nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-core/src/test/java/org/nuxeo/ecm/directory/core/CoreDirectoryInit.java

```diff
@@ -84,6 +99,17 @@ public void populate(CoreSession session) {
         user1.setProperty("schema1", "bar", "bar1");
         session.saveDocument(user1);
 
+        // Creates SHA1 passwords for unit test
+        ENC_PWD_USERSHA1 = PasswordHelper.hashPassword(DOC_PWD_USERSHA1, PasswordHelper.SSHA);
+        ENC_PWD_BADPWDSHA1 = PasswordHelper.hashPassword(DOC_PWD_BADPWDSHA1, PasswordHelper.SSHA);
```

Comment:
not sure this assignment is needed, is ENC_PWD_BADPWDSHA1 supposed to be checked later on in the test?